<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WikiInterface</class>
 <widget class="QWidget" name="WikiInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1463</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wiki &amp; Database Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Pivot" name="pivot_nav_bar" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacked_info_page">
     <widget class="QWidget" name="page_database_search">
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,3,7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="3">
        <widget class="BreadcrumbBar" name="bc_bar_item_list" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="StrongBodyLabel" name="label_3">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="LineEdit" name="line_edit_filter">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2" rowspan="2">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_description">
          <attribute name="title">
           <string>Description</string>
          </attribute>
          <attribute name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shows a description for the selected resource&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="text_browser_search_result">
             <property name="font">
              <font>
               <family>Dubai</family>
               <pointsize>11</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_details">
          <attribute name="title">
           <string>Details</string>
          </attribute>
          <attribute name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shows a detailed view of the selected resource's attributes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="TreeView" name="tree_view_json_data"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="ListView" name="list_view_api">
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_app_info">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="text_browser_app_info"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Pivot</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BreadcrumbBar</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListView</class>
   <extends>QListView</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
